Convert metric aggs docs runtime fields (#71260)
elastic/elasticsearch: docs/reference/aggregations/metrics/percentile-aggregation.asciidoc

==== Script

> If you need to run the aggregation against values not indexed you can run the
> aggregation on a <<runtime,runtime field>>. For example, if our load times
> are in milliseconds but we want percentiles calculated in seconds:

Reviewer: ```suggestion
If you need to run the aggregation against values that aren't indexed, use
a <<runtime,runtime field>>. For example, if our load times
are in milliseconds but you want percentiles calculated in seconds:
```
